src/main/java/games/strategy/engine/lobby/server/login/RsaAuthenticator.java

```diff
@@ -171,8 +175,21 @@ static String hashPasswordWithSalt(final String password) {
     return Collections.singletonMap(ENCRYPTED_PASSWORD_KEY, encryptPassword(challenge.get(RSA_PUBLIC_KEY), password));
   }
 
+  /**
+   * This method exists only to help support test.
+   * @deprecated Avoid calling this method, instead we should rework how this class is structured
+   */
```

Comment:
We should use both, the `@deprecated` comment in the javadocs and the '@Deprecated' Annotation on the method